<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>453</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="chatBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>聊天监控</string>
   </property>
   <widget class="QTextEdit" name="chatEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>19</y>
      <width>411</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="editBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>admin编辑框</string>
   </property>
   <widget class="QTextEdit" name="adminEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>19</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLine">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="titlelabel">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>0</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天服务器</string>
   </property>
  </widget>
  <widget class="QLabel" name="onlinelabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>在线：0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
